wiki/Tournaments/GTS/IGTS_2021/en.md

```diff
@@ -0,0 +1,636 @@
+
+### Stage regulations
+
+1. There are seven stages in the 1v1 tournament: the Qualifiers, the Group stage, the Round of 16, the Quarterfinals, the Semifinals, the Finals, and the Grand Finals.
+2. Teams will be seeded with the average of their ranks on each map during the Qualifiers round in 4 seeds: Top seed, High seed, Low seed, and Unseeded.
+3. Teams will be drawn in **8 groups of 4 teams** during live drawings which will happen on Sunday, 25 July 2021 (18:00 UTC).
+4. In the Group stage, all the teams from each group will face each other. The **best 2 teams** advance from their groups.
+5. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
+   1. Most matches won.
+   2. Highest "score" (number of beatmaps won - number of maps lost).
+   3. Most beatmaps won.
+   4. Winner of the match played previously between the tied teams.
+6. During the Group stage, winning 5:0 or having +1.0 score difference ratio will be considered a win by default.
+7. Each stage has the following win conditions:
+   - **Group Stage**: Best-of-9 (first to 5 points)
+   - **Round of 16, and the Quarterfinals**: Best-of-11 (first to 6 points)
+   - **Semifinals, the Finals, and the Grand Finals**: Best-of-13 (first to 7 points)
+
+### Qualifier regulations
+
+1. During this round there will be multiple lobbies available at various hours.
+2. A private match will be created and teams will be invited into the lobby when it is their time to play. Make sure to be online at this time.
+3. There will be no warmups during this stage.
+4. Failed scores will count.
+5. Only two players from each team will be playing per map. There are no limitations on player swapping.
+6. Seeding will be done with the average rank of each team on each map.
+7. Maps will be played twice in a row during the lobby. The best run will be taken into account for the ranking.
+8. Only the 32 best teams will continue on to the bracket stage.
+
+### Mappool instructions
+
+1. There will be one mappool for each stage.
+2. The loser's bracket will play on the same pool as the winner's bracket of the same weekend.
+3. The Qualifier pool will be different from all other rounds, as it will have a format of 2 NoMod maps, 1 Hidden map, 1 HardRock map, 1 DoubleTime map, and 1 FreeMod map.
+4. Each mappool consists of 6 sections: **NoMod, Hidden, HardRock, DoubleTime, FreeMod, and Tiebreaker**.
+5. Total maps per stage will be listed as follows:
+   - The Group stage mappool will consist of 14 maps.
+   - The Round of 16 and Quarterfinals mappools will consist of 15 maps.
+   - The Semifinals, Finals and Grand Finals mappools will consist of 16 maps.
+6. All stages will have 2 Hidden, HardRock, and DoubleTime maps, and 3 FreeMod maps.
+7. The FreeMod pool is played with FreeMod activated. Every player **has to pick** Hidden, HardRock, or both. At least one player of each team must have at least one mod activated.
```

Comment:
> does this mean that you *can't* use no mods (second sentence) or that you *can* use no mods (third sentence)?

mind checking this? if it's the former, then the third sentence should be removed, and if it's the latter, then it would look like this

```suggestion
7. The FreeMod pool is played with FreeMod activated. Every player may pick a combination of Hidden and HardRock, but at least one player of each team must have at least one mod activated.
```

Reply:
Should be latter, they can use No Mods.